### PR TITLE
enhancement(observability): update check-event script

### PR DESCRIPTION
**Pull request by jdrouet**
This PR removes the `skip check-event` that were remaining and fixes the bugs that were left.

### CODE REVIEW

**Review by spencergilbert — APPROVED** [review]

File: src/internal_events/unix.rs [hunk 1]
```diff
@@ -65,10 +76,19 @@ pub struct UnixSocketFileDeleteError<'a> {
...
             path = %self.path.display(),
             error = %self.error,
+            error_code = "delete_socket_file",
+            error_type = error_type::WRITER_FAILED,
+            stage = error_stage::PROCESSING,
+        );
+        counter!(
+            "component_errors_total", 1,
+            "error_code" => "delete_socket_file",
+            "error_type" => error_type::CONNECTION_FAILED,
```
Comment (jszwedko) [inline src/internal_events/unix.rs:90]: This `error_type` doesn't match the log.

File: src/internal_events/http_client.rs [hunk 2]
```diff
@@ -76,6 +75,13 @@ impl<'a> InternalEvent for GotHttpError<'a> {
         debug!(
```
[the remark this answers is not in the document]
Comment (jdrouet) [reply src/internal_events/http_client.rs]: fixed 😉

**Review by bruceg — COMMENTED** [review]
This is almost there for me. I have a couple more questions of the message parsing in the script.

File: scripts/check-events [hunk 3]
```diff
@@ -201,14 +208,14 @@ Find.find('.') do |path|
         block.scan(/
                     (trace|debug|info|warn|error)! # The log type
                     \(\s*(?:message\s*=\s*)? # Skip any leading "message =" bit
-                    "([^({)][^("]+)" # The log message text
+                    (?:"([^({)][^("]+)"|([^,]+)) # The log message text
```
[the remark this answers is not in the document]
Comment (jdrouet) [reply scripts/check-events:211]: we have this one for example: https://github.com/vectordotdev/vector/blob/master/src/internal_events/kubernetes_logs.rs#L132

File: scripts/check-events [hunk 4]
```diff
@@ -201,14 +208,14 @@ Find.find('.') do |path|
...
                     ([^;]*?) # Match the parameter list
                     \)(?:;|\n\s*}) # Normally would end with simply ");", but some are missing the semicolon
-                   /mx) do |type, message, parameters|
+                   /mx) do |type, raw_message, var_message, parameters|
           parameters = parameters.scan(/([a-z0-9_]+) *= .|[?%]([a-z0-9_.]+)/) \
                          .map { |assignment, simple| assignment or simple }
 
-          $all_events[event_name].add_log(type, message, parameters)
+          $all_events[event_name].add_log(type, raw_message, parameters)
```
[the remark this answers is not in the document]
Comment (jdrouet) [reply scripts/check-events]: yep, fixed that 😉

File: scripts/check-events (hunk 3 above)
[the remark this answers is not in the document]
Comment (bruceg) [reply scripts/check-events:211]: That is… unfortunate. Oh well.

**Review by bruceg — APPROVED** [review]

**Comment by jdrouet** [comment]
The failing windows test should be fixed by https://github.com/vectordotdev/vector/pull/12040

**Pull request merged**